### DISCUSSION TITLE
hello i could use some help

### DISCUSSION THREAD

**Discussion by Starlord-199** [navidrome/navidrome#3325] [Q&A]
hello I could use some help, I would like to ask you it gives me this error I am trying to instal navidrome on windows “I don't understand why it wasn't made a normal application where I would start the program is enough” comunqui I installed the program but when I go to start it I can't start it because it freezes. I thank everyone who will help me.

I attach what the command line tells me

<img width="1440" alt="Screenshot 2024-09-28 alle 16 16 25" src="https://github.com/user-attachments/assets/20f58cbd-12c1-40ff-ba44-7ba0028fea43">

**Comment by deluan** [comment]
I admit the process to install Navidrome on Windows is not for the faint of heart. There is a PR that adds a Setup Wizard and will most probably be available in the next release: #3125. The MSI from that PR is updated with the latest release, so give it a try.
